requirements_locked.txt

```diff
@@ -0,0 +1,55 @@
+#
```

Comment:
Why is this file useful?

Reply:
This is the lock file I use the install requirements in the CI - sorry that somehow didn't make it to this commit.